Tasks/GruntV0/Tests/L0.ts

```diff
@@ -1,16 +1,244 @@
 import fs = require('fs');
+import os = require('os');
 import assert = require('assert');
 import path = require('path');
+import * as ttm from 'azure-pipelines-task-lib/mock-test';
+
+const isWin = os.type().match(/^Win/);
 
 describe('GruntV0 Suite', function () {
-    before(() => {
+    this.timeout(parseInt(process.env.TASK_TEST_TIMEOUT) || 30000);
+
+    before((done) => {
```

Comment:
would you mind adding type annotation here?

Reply:
Yes, sure. Thanks!